app/src/main/res/layout/nav_header.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
+  android:layout_width="@dimen/navdrawer_width"
+  android:layout_height="@dimen/navdrawer_image_height"
+  android:orientation="vertical">
+
+  <ImageView
+    android:id="@+id/navdrawer_image"
+    android:layout_width="@dimen/navdrawer_width"
+    android:layout_height="@dimen/navdrawer_image_height"
+    android:contentDescription="@string/cd_navdrawer_image"
+    android:scaleType="centerCrop"
+    android:src="@drawable/ic_logo_wide"/>
+
+  <ImageView
```

Comment:
@tasomaniac user profile pic is getting displayed only if user does login. BTW, it's also essential to test the application without doing login :D